src/backend/singlenode/singlenode.cpp

```diff
@@ -62,6 +78,7 @@ namespace argo {
 			memory_size = size;
 			using namespace data_distribution;
 			naive_data_distribution<0>::set_memory_space(nodes, memory, size);
+			sig::signal_handler<SIGSEGV>::install_argo_handler(&singlenode_handler);
```

Comment:
Should a backend that doesn't use a signal handler install one in the first place? Isn't this just extra overhead?

Reply:
In principle you are right, this serves no "purpose". However, this is true of the entire singlenode backend, which you yourself suggested to be renamed to "dummy". For the purpose of a dummy, I believe the intention of this code is to test that the API is properly defined and teach how to apply it. Therefore I would argue against removing the code.

Reply:
OK, makes sense. This wasn't a blocking comment anyway.